src/BaseCommand.js

```diff
@@ -177,7 +193,11 @@ class BaseCommand extends Command {
 
 BaseCommand.flags = {
   verbose: Flags.boolean({ char: 'v', description: 'Verbose output' }),
-  version: Flags.boolean({ description: 'Show version' })
+  version: Flags.boolean({ description: 'Show version' }),
+  allowEventsTemplates: Flags.boolean({
```

Comment:
oh and 
3. if we use a flag we should use dashes instead of camelCase `allow-events-templates`

Reply:
@purplecabbage any thoughts on the approach to use here?

Reply:
`allow-events-templates` i can use this if we decide to go with the flag approach

Reply:
if its an env variable, how and where do i set it?

Reply:
made allow-events-templates flag local to the add event template command

Reply:
Definitely all flags should be kebab-case, thanks.

If the flag is temporary, we should never add it as a flag because it will break people in the future unless we bump a major version, and add a deprecation period - so we should try to avoid this as much as possible.

We haven't done this before but we could prefix `experimental-` to the flag name, much like how node does it for their experimental flags, which have no guarantees. Either using this prefix or the env var will break users once it is removed, but at least with the flag it will be more explicit.

Setting the flag to automatically pick up an env var is part of the oclif flag definition, see https://oclif.io/docs/flags (`env` key)

WDYT?

Reply:
sure, ill add the experimental- prefix to the flag.